articles/openshift/oshift-lifecycle.md

```diff
@@ -0,0 +1,50 @@
+---
+title: UKCloud for OpenShift lifecycle and patch management | UKCloud Ltd
+description: Provides an overview of how UKCloud manages the patching and lifecycle support of its OpenShift service
+services: openshift
+author: Steve Dixon
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenShift lifecycle and patch management
+toc_fullpath: openshift/oshift-lifecycle.md
```

Comment:
File name should be oshift-ref-lifecycle.md

Needs changing here and in the toc_mdlink metadata tag as well as the actual filename itself